stable/registry/templates/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
```

Comment:
Set `app: {{ template "name" . }}`. It allows better use of labels when querying, and is a pattern I'm pushing to adopt.